README.md

```diff
@@ -2,7 +2,13 @@
 This is the examples repository for the [strongDM Terraform Provider](https://github.com/strongdm/terraform-provider-sdm).
 
 ---
-> **NOTE:**
-> This repository is currently under construction. The content is expected to change in the coming weeks.
-
+> **NOTE:**  
+> To increase flexibility when managing thousands of Resources, Role Grants have
```

Comment:
I'd maybe just say large numbers of resources? Surely Access Rules are more flexible with even mediocre numbers, I don't think you have to have thousands to realize the benefits.